### PR TITLE
ref(js): Extract codeKeywords from codeBlock component

### PR DESCRIPTION
**Pull request by evanpurkhiser**
There's just too much stuff in here

### CODE REVIEW

File: src/components/codeKeywords.tsx [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+  codeKeywords?.PROJECT?.forEach(projectKeyword => {
+    orgSet.add(projectKeyword.ORG_SLUG);
+  });
+  const orgSlugs = [...orgSet];
+
+  const [isAnimating, setIsAnimating] = useState(false);
+
+  if (!codeKeywords.USER) {
+    // User is not logged in - show dummy token
+    return <Fragment>sntrys_YOUR_TOKEN_HERE</Fragment>;
```
Comment (shanamatthews) [inline src/components/codeKeywords.tsx:194]: unrelated to your change, but do you know why this is "sntrys_YOUR_TOKEN_HERE"? 
the "sntrys" parts specifically just looks like a typo

**Review by shanamatthews — APPROVED** [review]
Should maybe break this up further, but this is a good start

**Comment by evanpurkhiser** [comment]
Yeah I plan to split this up even more.

File: src/components/codeKeywords.tsx (hunk 1 above)
Comment (mydea) [reply src/components/codeKeywords.tsx:194]: this is correct, the org tokens _always_ have the `sntrys_` prefix, so you can identify immediately that this is a sentry token :)

**Pull request merged**